pylint/lint/run.py

```diff
@@ -77,6 +82,7 @@ def __init__(
                 args,
                 {
                     # option: (callback, takearg)
+                    "version": (display_version, False),
```

Comment:
Let's add it as callback like the rest of the methods in this list to keep it consistent.

Reply:
Done in https://github.com/PyCQA/pylint/pull/3581/commits/2571babfb9922a97ad47e07b0a8e3422f3d510ea